2019/CG-06.md

```diff
@@ -75,7 +82,20 @@ These are a few suggestions for accommodation including the distances to Igalia
     1. Find volunteers for note taking
     1. Adoption of the agenda
     1. Proposals and discussions
-       1. TBD
+       1. [Bulk Memory](https://github.com/WebAssembly/proposals/issues/18) (½ hr)
+         1. Passive element segment encoding
+       1. [Multi Value](https://github.com/WebAssembly/proposals/issues/19) (update) (¼-½ hr)
+         1. Poll: phase 4
```

Comment:
Requires two or more Web VMs that implement.  Which ones?  Also a toolchain.  Which?  (I ask because I don't know.  There's an in-progress implementation for Firefox but nothing's landed at this point.)

Reply:
I think V8 and WABT supports multi values. SpiderMonkey does not support it. Chakra has stop development of new features.

Reply:
I thought SM was complete. I know toolchain support is moving forward but not quite there either. So perhaps this is a bit premature.